x-pack/plugin/inference/qa/rolling-upgrade/build.gradle

```diff
@@ -0,0 +1,37 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0 and the Server Side Public License, v 1; you may not use this file except
+ * in compliance with, at your election, the Elastic License 2.0 or the Server
+ * Side Public License, v 1.
+ */
+
+import org.elasticsearch.gradle.Version
+import org.elasticsearch.gradle.VersionProperties
+import org.elasticsearch.gradle.internal.info.BuildParams
+import org.elasticsearch.gradle.testclusters.StandaloneRestIntegTestTask
+
+apply plugin: 'elasticsearch.internal-java-rest-test'
+apply plugin: 'elasticsearch.internal-test-artifact-base'
+apply plugin: 'elasticsearch.bwc-test'
+
+
+dependencies {
+  compileOnly project(':x-pack:plugin:core')
+  javaRestTestImplementation(testArtifact(project(xpackModule('core'))))
+  javaRestTestImplementation project(path: xpackModule('inference'))
+  clusterPlugins project(':x-pack:plugin:inference:qa:test-service-plugin')
+  javaRestTestImplementation(testArtifact(project(":qa:rolling-upgrade"), "javaRestTest"))
+}
+
+assert Version.fromString(VersionProperties.getVersions().get("elasticsearch")).getMajor() == 8:
```

Comment:
Just curious, does this mean once elasticsearch moves to version 9 it'll fail to remind us to create the migration tests?

Reply:
Yeah, what's the expected mitigation here? What needs to be done when we bump to 9.0?

Reply:
Copy and pasted without thinking. I've removed the assertion now